### PR TITLE
remove errant ` in git clone call

### PR DESCRIPTION
**Pull request by mrjones-plip**

### CODE REVIEW

**Comment by mrjones-plip** [comment]
oh - hah!  Thanks @eljhkrr  - but I guess you and I don't have write access so the approval doesn't allow me to merge.  I'll see who does :eye:

**Review by jkuester — APPROVED** [review]

**Pull request merged**
